docs/attributes-registry/k8s.md

```diff
@@ -20,6 +20,7 @@
 | `k8s.namespace.name` | string | The name of the namespace that the pod is running in. | `default` | Recommended |
 | `k8s.node.name` | string | The name of the Node. | `node-1` | Recommended |
 | `k8s.node.uid` | string | The UID of the Node. | `1eb3a0c6-0477-4080-a9cb-0cb7db65c6a2` | Recommended |
+| `k8s.pod.labels.<key>` | string | The labels placed on the Pod, the `<key>` being the label name, the value being the label value. | `k8s.pod.labels.app=my-app`; `k8s.pod.labels.mycompany.io/arch=x64`; `k8s.pod.labels.data=` | Recommended |
```

Comment:
Should we include annotations `k8s.pod.annotations.<annotation key>`? While labels contain identifying info which is useful to select/group pods by, annotations will have more non-identifying supplementary info that is usually read or added by external tooling. Some e.g. info found in annotations for e.g., custom config used by controllers such as Istio, build info, details about the team which owns the service, etc. I think we should make note of annotations being a potential source of attrs for k8s pods with requirement level = Opt-in. Doesn't have to be in this PR, just want to see what others think about annotations. Fwiw, the k8sattributes processor can also extract annotations as attributes.

Reply:
We should definitely have the same thing for annotations (and I'd even vote for `Recommended` requirement level).  But I agree that we should handle that discussion in a different PR.